src/ServicePulse.Host/vue/src/components/monitoring/LargeGraph.vue

```diff
@@ -0,0 +1,176 @@
+<script setup lang="ts">
+import { ref, computed } from "vue";
+import { useFormatTime, useFormatLargeNumber } from "../../composables/formatter";
+import { useGraph } from "./graphLines";
+import type { PlotData } from "./PlotData";
+
+const props = defineProps<{
+  isdurationgraph: boolean;
+  metricsuffix?: string | undefined;
```

Comment:
@PhilBastian, you don't actually need the ` | undefined`, because you have declared the prop as optional `?`.
So when you declare a prop as optional it will add automatically ` | undefined`.
But if you want to also support `null`, then you would include it, example:
```ts
 myVar?: string | null; // this will be `string | undefined | null`
```
I hope it makes sense?
BTW, I think the code base tends to use camelCase, in my IDE this code is highlighted as a spelling mistake.
![image](https://github.com/Particular/ServicePulse/assets/122651/048dd670-c0f7-4974-8644-8fce9f1d8e86)

Reply:
yep, was there is previous code so i didn't change it